packages/react-day-picker/src/contexts/Modifiers/ModifiersContext.test.tsx

```diff
@@ -0,0 +1,15 @@
+describe('when passing a "selected" custom modifier', () => {
+  test('they should replace the internal "selected" modifiers');
```

Comment:
Do we want to support that behavior? I kind of see “selected” as something you should never set separately from the selected prop